libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5759,6 +5760,16 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
             camera->send_camera_settings(chan);
         }
         break;
+        case MSG_CAMERA_CAPTURE_STATUS:
+        {
+            AP_Camera *camera = AP::camera();
+            if (camera == nullptr) {
+                break;
+            }
+            CHECK_PAYLOAD_SIZE(CAMERA_CAPTURE_STATUS);
```

Comment:
Not a blocker but it's odd that we check this payload-size here because this assumes we are only sending one message while there could actually be one sent per backend.

I suspect this is an existing issue for how we send other messages as well.